src/main/java/org/folio/dataimport/util/Try.java

```diff
@@ -0,0 +1,54 @@
+package org.folio.dataimport.util;
+
+import io.vertx.core.Future;
+
+import java.util.function.Supplier;
+
+/**
+ * Util class which contains methods with boiler-plate code for exceptions handling under async methods calls
+ */
+public class Try {
+
+  private Try() {
+  }
+
+  /**
+   * Return future with result returned by specified task.
+   * If the {@code task} throws an exception, the returned future will be failed with this exception
+   *
```

Comment:
I sugest:
Executes a task and returns either a future with result or a future with an exception